src/home/components/Home.js

```diff
@@ -67,7 +74,13 @@ const StoryMaps = ({ storyMaps, fetching }) => {
     return <StoryMapsHomeCardDefault />;
   }
 
-  return <StoryMapsHomeCard storyMaps={storyMaps} />;
+  return (
+    <StoryMapsCard
+      onDeleteSuccess={onDeleteSuccess}
+      storyMaps={storyMaps}
+      title={title}
```

Comment:
Suggestion: I think you can pass here directly `t('storyMap.home_title')` instead of passing it from the parent component
```suggestion
      title={t('storyMap.home_title')}
```